docs/integrations/spark/spark.md

```diff
@@ -132,7 +132,7 @@ spark-submit --conf spark.driver.extraJavaOptions=-javaagent:<marquez-spark-jar-
 
 ### From Airflow
 The same parameters passed to `spark-submit` can be supplied from Airflow and other schedulers. If
-using the [marquez-airflow](airflow.md) integration, each task in the DAG has its own Run id
+using the [marquez-airflow](../airflow.md) integration, each task in the DAG has its own Run id
```

Comment:
marquez-airflow -> openlineage-airflow

Reply:
And fix all old marquez references 🙂 

Might be done in separate PR though.

Reply:
I was planning another PR to refurbish whole `spark.md`